cmd/kubeadm/app/constants/constants.go

```diff
@@ -425,8 +425,8 @@ var (
 		14: "3.3.10",
 		15: "3.3.10",
 		16: "3.3.15-0",
-		17: "3.3.17-0",
```

Comment:
Do we want to change default of 16 to 3.3.17-0?

Reply:
Make sense. ref: #83968

Reply:
I'm in favor of updating to same patch version for all k8s versions using 3.3 in kubeadm.